packages/ui-components/src/components/windowedlist.ts

```diff
@@ -733,29 +734,48 @@ export class WindowedList<
    *
    * @param options Constructor options
    */
-  constructor(options: WindowedList.IOptions<T>) {
-    // TODO probably needs to be able to customize outer HTML tag (could be ul / ol / table, ...)
+  constructor(options: WindowedList.IOptions<U, T>) {
```

Comment:
Could you avoid switching the template `U, T` to ease understanding (because U becomes T what is confusing).

Reply:
You're right. I was unhappy about this at the time. I'll look at this again.